lib/roo_on_rails/routemaster/lifecycle_events.rb

```diff
@@ -0,0 +1,49 @@
+require 'active_support/concern'
+require 'new_relic/agent'
```

Comment:
Missing a require for `roo_on_rails/routemaster/publishers` which is referenced.